packages/matchbox/src/components/ComboBox/ComboBoxTextField.js

```diff
@@ -85,11 +86,25 @@ function ComboBoxTextField(props) {
         {selectedItems.length > 0 && (
           <Box display="flex" pl="200" pt="0.375rem">
             <Inline space="100">
-              {selectedItems.map((item, i) => (
-                <Tag key={i} onRemove={!disabled ? () => removeItem(item) : null}>
-                  {itemToString(item)}
-                </Tag>
-              ))}
+              {selectedItems.map((item, i) => {
+                return (
+                  <span key={i}>
+                    <Tag onRemove={!disabled ? () => removeItem(item) : null}>
+                      {itemToString(item)}
+                    </Tag>
+                    {delimiter && i < selectedItems.length - 1 ? (
+                      <Box
+                        as="span"
+                        ml="100"
+                        fontWeight="medium"
+                        style={{ textTransform: 'uppercase', fontStyle: 'italic' }}
```

Comment:
Doesn't really matter either way, but `fontStyle` is a prop available on `Box` so could do that instead of assigning it here